config/routes.rb

```diff
@@ -3,14 +3,16 @@
 
   ActiveAdmin.routes(self)
 
-  mount News::Engine => "/news"
+  scope '(:locale)', locale: /#{I18n.locale}/ do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.